pkg/test/framework/components/istio/flags.go

```diff
@@ -65,4 +65,8 @@ func init() {
 		settingsFromCommandline.EgressGatewayIstioLabel,
 		`Specifies the istio label of the egressgateway to search for when running tests in a preinstalled istio installation.
 		Should only be set when istio.test.kube.deploy=false`)
+	flag.StringVar(&settingsFromCommandline.SharedMeshConfigName, "istio.test.kube.sharedMeshConfigName",
+		settingsFromCommandline.SharedMeshConfigName,
+		`Specifies the name of the SHARED_MESH_CONFIG defined and created by the user upon installing Istio.
+		Should only be set when istio.test.kube.userSharedMeshConfig=true and istio.test.kube.deploy=false.`)
```

Comment:
`istio.test.kube.userSharedMeshConfig=true` still needs to be implemented? Or should this be removed?

Reply:
Just checked with Niranjan, `istio.test.kube.userSharedMeshConfig=true` should be removed, just `SharedMeshConfigName` is good enough.